pkg/controllers/binding/common.go

```diff
@@ -116,6 +111,13 @@ func ensureWork(
 			}
 		}
 
+		// Call ReviseReplica func first, as ApplyOverridePolicy has the highest priority
```

Comment:
This comment is a bit ambiguous, as it refers to the `ReviseReplica` function after it has already been called, I suggest adjusting it:

```suggestion
		// We should call ApplyOverridePolicies last, as override rules have the highest priority
```

Reply:
updated